Source/WebInspectorUI/UserInterface/Views/AuditTestContentView.css

```diff
@@ -134,20 +134,20 @@
 }
 
 .content-view.audit-test > header table.controls > tr.supports .warning {
-    margin-inline-start: 4px;
+    margin-inline-start: .25em;
     color: var(--text-color-secondary);
 }
 
 .content-view.audit-test > header table.controls > tr.supports .warning:not(:empty)::before {
     display: inline-block;
     width: 1em;
-    margin-inline-end: 2px;
-    vertical-align: -1px;
+    margin-inline-end: .125em;
+    vertical-align: -0.0625em;
```

Comment:
Won't the negative relative size potentially cause the contents to moving farther outside the element's own box as the font size increases?

Reply:
It may not if there's a larger negative corresponding margin, but I wasn't sure if that had been incorporated elsewhere.

Reply:
After testing both of the vertical-aligns as one positive and one negative or both, the best solution was to keep this vertical-align positive and the other vertical-align negative to keep them aligned.